src/Maker/MakeEntity.php

```diff
@@ -42,12 +42,10 @@
 {
     private $fileManager;
     private $doctrineHelper;
-    private $projectDirectory;
 
     public function __construct(FileManager $fileManager, string $projectDirectory, DoctrineHelper $doctrineHelper)
```

Comment:
This one cannot be removed because this class isn't `@internal` and  `$projectDirectory` isn't the last argument.